src/nyc_trees/apps/home/training/__init__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django_tinsel.decorators import route, render_template
+from django_tinsel.utils import decorate as do
+
+from apps.home.training.types import TrainingStep, TrainingGateway
+from apps.home.training import views as v
+
+# This package attempts to keep all the particulars of training encapsulated,
+# such that you can modify how things are wired together in this file without
+# digging into the implementation. There are some key exceptions to this:
+# * apps.home.urls - imports from this file, gets url params
+# * apps.core.models
+#   - the user class does a delayed import for a convenience method
+#   - the user class has carefully named booleans corresponding
+#     to training steps. modify with care.
+
+getting_started = TrainingStep('getting_started',
+                               'Getting Started', '20 minutes')
+the_mapping_method = TrainingStep('the_mapping_method',
+                                  'The Mapping Method', '20 minutes')
+tree_data = TrainingStep('tree_data',
+                         'Tree Data', '20 minutes')
+tree_surroundings = TrainingStep('tree_surroundings',
+                                 'Tree Surroundings', '40 minutes')
+groups_to_follow = TrainingStep('groups_to_follow',
+                                'Groups To Follow', '5 minutes',
+                                route(GET=do(
+                                    render_template(
+                                        'home/groups_to_follow.html'),
+                                    v.groups_to_follow)))
+
+training_list_page = route(GET=do(render_template('home/training.html'),
+                                  v.training_list_page))
+
+training_summary = TrainingGateway(training_list_page,
+                                   getting_started, groups_to_follow)
+
+getting_started.previous_step = training_summary
+getting_started.next_step = the_mapping_method
+the_mapping_method.previous_step = getting_started
+the_mapping_method.next_step = tree_data
+tree_data.previous_step = the_mapping_method
+tree_data.next_step = tree_surroundings
+tree_surroundings.previous_step = tree_data
+tree_surroundings.next_step = groups_to_follow
+groups_to_follow.previous_step = tree_surroundings
+groups_to_follow.next_step = training_summary
```

Comment:
Can't do that in this case, `next_step` must be a step-like object in order for the current step (groups_to_follow) to have a destination to bind its `mark_progress` endpoint to.